travvy/src/components/AttractionDetails.vue

```diff
@@ -3,31 +3,47 @@
     <!-- Button to go to go back to the AttractionsList page -->
     <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
 
-    <h1>{{$store.state.recommendedAttractions[0].attraction_name}} </h1>
+    <h1>{{$store.state.recommendedAttractions[id].attraction_name}} </h1>
     <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
     <br>
     <h1>Description</h1>
     <p>The CN Tower is a 553.3 m-high concrete communications and observation tower located in Downtown Toronto, Ontario, Canada.</p>
-    <h2>Address: <span>{{$store.state.recommendedAttractions[0].street_address}}</span> </h2>
-    <h3>Email: {{$store.state.recommendedAttractions[0].email_address}}</h3>
-    <h3>Phone Number: {{$store.state.recommendedAttractions[0].phone_number}}</h3>
-    <h3>Ticket Pricing: {{$store.state.recommendedAttractions[0].ticket_price}}</h3>
-    <h3>Maximum Group Size: {{$store.state.recommendedAttractions[0].max_number_guest}}</h3>
-    <h3>Are masks required: {{$store.state.recommendedAttractions[0].mask_required}}</h3>
-    <h3>COVID-19 Safety Measures: {{$store.state.recommendedAttractions[0].covid_safety_measures}}</h3>
+    <h2>Address: <span>{{$store.state.recommendedAttractions[id].street_address}}</span> </h2>
+    <h3>Email: {{$store.state.recommendedAttractions[id].email_address}}</h3>
+    <h3>Phone Number: {{$store.state.recommendedAttractions[id].phone_number}}</h3>
+    <h3>Ticket Pricing: {{$store.state.recommendedAttractions[id].ticket_price}}</h3>
+    <h3>Maximum Group Size: {{$store.state.recommendedAttractions[id].max_number_guest}}</h3>
+    <h3>Are masks required: {{$store.state.recommendedAttractions[id].mask_required}}</h3>
+    <h3>COVID-19 Safety Measures: {{$store.state.recommendedAttractions[id].covid_safety_measures}}</h3>
```

Comment:
Thanks for making these dynamic - I think it'll be a great addition to our site!